Seminars/Homeworks/Homework_001/Task_001.py

```diff
@@ -4,14 +4,10 @@
 # - 7 -> да
 # - 1 -> нет
 
-
-
-
 day_number = int(input())
 if day_number <= 5 > 0:
```

Comment:
Михаил, вы в условии справа сравниваете пятерку с нулем. Разумеется, пятерка всегда будет больше нуля. Надо было сделать так:
```suggestion
if 0 < day_number <= 5:
```
Другое условие исправьте сами.